lib/sandbox/execution-location.js

```diff
@@ -0,0 +1,17 @@
+class ExecutionClass extends Array {
```

Comment:
The name for the class is weird. Why not call it `ExecutionLocation` itself?

Reply:
Changed